bin/rakudobrew

```diff
@@ -57,13 +56,22 @@ if ($arg eq 'switch') {
         say "Not running anything at the moment. Use 'rakudobrew switch' to set an implementation";
     }
 } elsif ($arg eq 'build') {
-    my ($impl, $ver) = (shift, shift);
+    my $impl = shift;
+    my $ver = shift
+        if $ARGV[0] !~ /^--/;
+    my $configure_opts;
+    if ( $ARGV[0] =~ /^--configure-opts=/ ) {
```

Comment:
Please stick to the coding style; nowhere else in rakudobrew there are spaces in if conditions :)